src/duplicacy_snapshotmanager.go

```diff
@@ -1017,7 +1017,7 @@ func (manager *SnapshotManager) ShowStatisticsTabular(snapshotMap map[string][]*
 			}
 		}
 		fmt.Fprintln(tableWriter, fmt.Sprintf(
```

Comment:
I think you want:
~~~
"%s \tall \t \t \t \t \t \t%d \t%s \t%d \t%s \t",
snapshotID, uniqueChunkCount, PrettyNumber(uniqueChunkSize), totalChunkCount, PrettyNumber(totalChunkSize)))
~~~

Reply:
I was thinking here that the same applies to the sum of "bytes", which is the sum of the last column

Then it would look like this:

```
snap | rev |                          | files |    bytes | chunks |    bytes | uniq |    bytes | new |    bytes |
test |   1 | @ 2018-07-02 11:36 -hash |     4 | 694,124K |    603 | 690,880K |    3 |      39K | 603 | 690,880K |
test |   2 | @ 2018-07-02 11:38       |     5 | 694,679K |    604 | 691,366K |    3 |      39K |   4 |     525K |
test |   3 | @ 2018-07-02 11:39       |     6 | 695,235K |    604 | 691,366K |    3 |      39K |   3 |      39K |
test |   4 | @ 2018-07-02 11:39       |    11 | 698,011K |    605 | 693,793K |    3 |      40K |   4 |   2,466K |
test |   5 | @ 2018-07-02 11:40       |    11 | 698,011K |    607 | 695,046K |    5 |   1,293K |   5 |   1,293K |
test | all |                          |       |          |        |          |      |          | 619 | 695,205K |
```

And of code would then be:

`"%s \tall \t \t \t \t \t \t \t \t%d \t%s \t"`